scenarioo-server/build.gradle

```diff
@@ -96,55 +96,47 @@ dependencies {
 }
 
 
-task createVersionPropertiesFile {
-    doLast {
-        def aggregatedDataFormatVersion = project.scenariooAggregatedDataFormatVersion
-        def apiVersion = project.scenariooApiVersion
-        def documentationVersion = project.documentationVersion
-
-        File versionFile = new File(sourceSets.main.output.classesDirs.getSingleFile().absolutePath + '/version.properties');
-        versionFile.write('version=' + versionWithGitCommit + '\n' +
-                'build-date=' + new Date() + '\n' +
-                'apiVersion=' + apiVersion + '\n' +
-                'aggregatedDataFormatVersion=' + aggregatedDataFormatVersion + '\n' +
-                'documentationVersion=' + documentationVersion + '\n')
+/**
+ * Replace properties in important resource files
```

Comment:
I think java doc style should only be used on methods. I would use a /* single comment */ style here.